processor.js

```diff
@@ -16,55 +16,74 @@ var notifier_messager = require("./messager/messager")({
   queueUrl: process.env.OUTGOING_QUEUE_URL
 });
 
-var mailroom = require('webmaker-mailroom')();
+var rateLimitConfig,
+    ttl = process.env.TTL || 1000 * 60 * 5;
 
-var workers = async.applyEachSeries([
-  worker.archiver(archiver_config),
-  worker.backwards_compatibility,
-  worker.remind_user_about_event(notifier_messager, mailroom),
-  worker.login_request(notifier_messager, mailroom),
-  worker.send_sms(notifier_messager),
-  worker.receive_coinbase_donation(notifier_messager, mailroom),
-  worker.reset_request(notifier_messager, mailroom),
-  worker.send_event_host_email(notifier_messager, mailroom),
-  worker.send_mofo_staff_email(notifier_messager, mailroom, process.env.MOFO_STAFF_EMAIL),
-  worker.send_new_user_email(notifier_messager, mailroom),
-  worker.event_mentor_confirmation_email(notifier_messager, mailroom),
-  worker.event_coorganizer_added(notifier_messager, mailroom),
-  worker.sign_up_for_bsd(notifier_messager),
-  worker.badge_awarded_send_email(notifier_messager),
-  worker.badge_application_denied(notifier_messager, mailroom),
-  worker.hive_badge_awarded(notifier_messager, mailroom),
-  worker.suggest_featured_resource(notifier_messager, process.env.SFR_SPREADSHEET, process.env.SFR_WORKSHEET)
-]);
+if ( process.env.REDIS_HOST && process.env.REDIS_PORT ) {
```

Comment:
Instead of keying based on the presence of the `REDIS_HOST`, we should have a `CACHE_ENGINE` option that's `memory` or `redis` in our case, and then setup cache options based on that